ao/test_number_of_substrings_with_only_1s.py

```diff
@@ -21,6 +21,9 @@ def test_eg3(self):
     def test_eg4(self):
```

Comment:
Adding a test case description. This would benefit the SO for documentation and for future maintainers.